app/models/immunisation_import_row.rb

```diff
@@ -253,7 +253,8 @@ def school
   end
 
   def programme
-    @programme ||= programmes_by_name[programme_name&.to_s]
+    @programme ||=
+      programmes_by_name[programme_name&.to_s] || vaccine&.programme
```

Comment:
Does this change our spec for the imports? Just wondering if we've considered the wider implication of making what seems like a small improvement.

Reply:
I'm not too worried about it because the priority is given to if the programme as provided explicitly in the files if it's present (we only rely on the vaccine as a fallback). I've also added a validation to check the vaccine is relevant for the programme if specified.